arrocke/gloss-translation
# feat: preview verses in lexicon

## What has changed



## Connected Issues

closes #336 
## QA Steps

- Either run `nx run db:prisma migrate reset` to reset your DB to the seed file, or run `nx import-lsj db` and `nx import-bdb db` to update the lexicon entries in your DB.
- Go to any verse in the translation screen of the frontend
  - Example: `Matthew 1:3`
- Click on an original language word to open the sidebar
  - Example: `Ἑσρώμ;`
- Click any reference links you see in the sidebar
  - Reference links are in green text
- Verify that the verse preview is loaded correctly for the select reference


NOTE - there may be certain edge cases where the references aren't parsed or handled correctly. Getting the parsing logic correct is tricky. It might be nice to save those remaining edge cases for a separate issue and/or PR, since this one has dragged on so long. I'm pretty sure this PR covers at least 90% of the references correctly.



## Post-Deployment



- [ ] Update lexicons in production DB
